app/views/home/index.html.erb

```diff
@@ -1,30 +1,28 @@
-<section class="hero is-white is-halfheight">
+<section class="hero is-small is-white is-halfheight">
   <div class="hero-body">
     <div class="container">
       <div class="columns is-vcentered reverse-columns">
         <div class="column
-                    is-10-mobile is-offset-1-mobile
+                    is-10-mobile is-offset-0-mobile
                     is-10-tablet is-offset-1-tablet
                     is-5-desktop is-offset-1-desktop
                     is-5-widescreen is-offset-1-widescreen
-                    is-5-fullhd is-offset-1-fullhd">
-          <h1 class="title titled is-1 mb-6">
-            Children wallet
+                    is-5-fullhd ml-6-pmoney-fullhd">
+          <h1 class="title titled is-1 mb-6 has-text-pmoney-color is-size-3-mobile">
```

Comment:
Can we change the css variable so that main text is not black like now but the new one?

Reply:
Done.